docs/01-getting-started.md

```diff
@@ -44,7 +32,20 @@ you use `sleeper`.
 
 You can also upgrade the CLI to a different version with `sleeper cli upgrade`.
 
-#### Authentication
+## Deploy to Docker
+
+The quickest way to get an instance of Sleeper is to deploy to LocalStack in Docker on your local machine. See the
+documentation on [deploying to localstack](10-deploy-to-localstack.md) for more information.
```

Comment:
Can we emphasise that the LocalStack option is limited? Add "Note that the LocalStack version has very limited functionality in comparison to the AWS version, and can only handle small volumes of data." between these two sentences.